### DISCUSSION TITLE
Addition - Kobra 2 Neo Stock Control Unit support for KlipperScreen

### DISCUSSION THREAD

**Discussion by soosp** [1coderookie/Kobra2NeoInsights#10] [General]
It seems that there is KlipperScreen support for Stock Control Unit. See: [https://github.com/cheadrian/kobra2neo-klipper/tree/main/KlipperScreen](url) and [https://github.com/jokubasver/Anycubic-Kobra-Go-Neo-LCD-Driver?tab=readme-ov-file#hardware-setup](url)

**Comment by 1coderookie** [comment]
Thanks for reaching out (and sorry for my late reply!), I didn't come across the solution with the working encoder yet. 
Are you using it?
Since it seems to be based on @jokubasver solution, I wonder if it would work with the Kobra Go&Neo as well.  

(Btw: I turned it into a discussion since it's not an issue.. ;) )
